frontend/src/app/document-view/document-view.component.ts

```diff
@@ -68,7 +68,7 @@ export class DocumentViewComponent implements OnChanges {
     }
 
     isUrlField(field: CorpusField) {
-        return field.name === 'url' || field.name.startsWith('url_');
+        return field.displayType === 'url';
```

Comment:
This is unrelated to this PR?

Reply:
It's somewhat related. The URL display type was needed for the export because the JSON schema has that option, but the database did not.